src/main/java/com/aorri2/goodsforyou/common/exception/ErrorCode.java

```diff
@@ -10,7 +10,10 @@ public enum ErrorCode {
 	NOT_MATCHED_EMAIL("이메일이 일치하지 않습니다."),
 
 	//Product Error Code
-	NOT_EXIST_PRODUCT("해당 상품은 존재하지 않습니다.");
+	NOT_EXIST_PRODUCT("해당 상품은 존재하지 않습니다."),
+
+	//Transaction Error Code
+	UNAVAILABLE_PRODUCT("이미 판매 완료된 상품 입니다.");
```

Comment:
Unavailable 은 너무 포괄 적인 것 같아요.
판매가 완료된 상품 외에도 unanvailable 을 사용 할 수 있는 상품은 많을 것 같은데 조금 더 구체적이 이름이 좋지 않을까요??

Reply:
```suggestion
	ALREADY_SOLD_PRODUCT("이미 판매 완료된 상품 입니다.");
```
위와 같이 좀더 구체적인 이름으로 변경토록 해보겠습니다!